src/components/Quote.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+function Quote() {
+  const url = 'https://api.api-ninjas.com/v1/quotes?category=';
+  const category = 'knowledge';
+  const apiKey = 'iSOePepz25QMz4H2176HwA==Ys7JZLwntIHsSbKO';
+
+  const [data, setData] = React.useState('loadng quote...');
+
+  React.useEffect(() => {
+    const apiCall = async () => {
+      await fetch(url + category, {
+        headers: {
+          'X-Api-Key': apiKey,
+        },
+      })
+        .then((response) => response.json())
+        .then((data) => setData(data[0].quote))
+        .catch((error) => console.log(error));
```

Comment:
- [ ] It would be better to have three states in your quotes components one to return Quotes, a loading state, and an error state.  If Something went wrong! you can display the error message, if the quote is still loading you display loading  or return quotes if successful 👍🏿 You are currently using the same state to return the quotes or display the loading state please update your code to have different states  👍🏿